### PR TITLE
MGMT-2222 Delete all new lines from ssh key

### PR DESCRIPTION
**Pull request by tsorya**

### CODE REVIEW

File: internal/bminventory/inventory_test.go [hunk 1]
```diff
@@ -1266,14 +1266,20 @@ var _ = Describe("cluster", func() {
 					"AC61aG2a/8PFS08iBrKiMa48kn+hKXC4G4D5gj/QzIAgzWSl2tEzGQSoIVTucwOAL/jox2dmAa0RyKsnsHORppanuW4qD7KAcmas1GHrAq" +
 					"IfNyDiU2JR50r1jCxj5H76QxIuM= root@ocp-edge34.lab.eng.tlv2.redhat.com gggggggg fdddddddddddddddddddddddd" +
 					"dddddddddddddddd"
-				sshKeyWithNewLine := sshKey + " \n"
+				sshKeyWithNewLines := "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDi8KHZYGyPQjECHwytquI3rmpgoUn6M+lkeOD2nEKvYElLE5mPIeqF0izJIl56u" +
+					"ar2wda+3z107M9QkatE+dP4S9/Ltrlm+/ktAf4O6UoxNLUzv/TGHasb9g3Xkt8JTkohVzVK36622Sd8kLzEc61v1AonLWIADtpwq6/GvH" +
+					"MAuPK2R/H0rdKhTokylKZLDdTqQ+KUFelI6RNIaUBjtVrwkx1j0htxN11DjBVuUyPT2O1ejWegtrM0T+4vXGEA3g3YfbT2k0YnEzjXXqng" +
+					"qbXCYEJCZidp3pJLH/ilo4Y4BId/bx/bhzcbkZPeKlLwjR8g9sydce39bzPIQj+b7nlFv1Vot/77VNwkjXjYPUdUPu0d1PkFD9jKDOdB3f" +
+					"AC61aG2a/8PFS08iBrKiMa48kn+hKXC4G4D5gj/QzIAgzWSl2tEzGQSoIVTucwOAL/jox2dmAa0RyKsnsHORppanuW4qD7KAcmas1GHrAq" +
+					"IfNyDiU2JR50r1jCxj5H76QxIuM= root@ocp-edge34.lab.eng.tlv2.redhat.com\n gggggggg \nfdddddddddddddddddddddddd" +
```
Comment (filanov) [inline internal/bminventory/inventory_test.go:1274]: But it's still an invalid ssh key right?

Comment (tsorya) [reply internal/bminventory/inventory_test.go:1274]: it will be valid after removing \n

**Comment by avishayt** [comment]
Looks OK to me, @filanov do you have some concern?

**Comment by filanov** [comment]
> Looks OK to me, @filanov do you have some concern?

No it's fine, but lets way until after the release

File: internal/bminventory/inventory_test.go (hunk 1 above)
Comment (RazRegev) [reply internal/bminventory/inventory_test.go:1274]: it shouldn't be valid after removing the \n, https://github.com/openshift/assisted-service/pull/412

**Pull request closed without merging**